core/src/main/java/org/apache/calcite/sql2rel/SqlToRelConverter.java

```diff
@@ -1778,8 +1789,14 @@ public RelNode convertToSingleValueSubq(
     final SqlOperator comparisonOp;
 
     if (op instanceof SqlQuantifyOperator) {
-      comparisonOp = RelOptUtil.op(((SqlQuantifyOperator) op).comparisonKind,
-          SqlStdOperatorTable.EQUALS);
+      SqlQuantifyOperator quantifyOperator = (SqlQuantifyOperator) op;
+      if (quantifyOperator.comparisonKind == SqlKind.LIKE && !quantifyOperator.caseSensitive) {
+        // Special code for case insensitive like.
+        comparisonOp = SqlLibraryOperators.ILIKE;
+      } else {
+        comparisonOp = RelOptUtil.op(quantifyOperator.comparisonKind,
+            SqlStdOperatorTable.EQUALS);
+      }
```

Comment:
I think this is the only actual change to this file, the rest is formatting. If there is another change I need to review, please let me know.

Reply:
This was the only one. I'm not really sure why IntelliJ feels the need to reformat this entire file when I touch it. I set the options to the ones for Calcite but I guess it'll just do what it does.